Hunter/Assets/Scripts/View/View.cs

```diff
@@ -89,6 +85,9 @@ private void Update()
     {
         _game.Update();
 
+        Controller.HunterControler();
+        Controller.MousePosition();
```

Comment:
Is it right way to connect View and Controller with static methods?

Reply:
Well, We can make it the same way as _game - using instance. Also it might be better, if these two methods will be in another one (Like "ReadMoves")